InterfacePython/TasmanianSG.py

```diff
@@ -112,6 +112,7 @@
 pLibTSG.tsgGetInterpolationWeights.argtypes = [c_void_p, POINTER(c_double)]
 pLibTSG.tsgGetInterpolationWeightsStatic.argtypes = [c_void_p, POINTER(c_double), POINTER(c_double)]
 pLibTSG.tsgLoadNeededPoints.argtypes = [c_void_p, POINTER(c_double)]
+pLibTSG.tsgLoadNeededValues.argtypes = [c_void_p, POINTER(c_double)]
```

Comment:
As I think about this, we don't need this one. In Python, we only need:
```
def loadNeededValues(self, llValues):
    self.loadNeededPoints(llValues)
```

Keep the C variant, it can be useful for other purposes.

Reply:
Done.